rgw/v2/tests/s3_swift/test_rgw_orphan_list.py

```diff
@@ -0,0 +1,222 @@
+"""
+Validates RGW integrity by checking for orphan objects and data inconsistencies.
+
+This script uses 'rgw-orphan-list', 'radosgw-admin', and 'rados' to verify:
+1.  Empty RGW garbage collection.
+2.  No orphan data in 'default.rgw.buckets.data'.
+3.  Matching index counts for buckets in 'default.rgw.buckets.index'.
+
+Fails if orphans are found or index counts mismatch. Provides detailed logging and supports remote execution.
+"""
+
+import argparse
+import json
+import logging
+import os
+import subprocess
+import sys
+import traceback
+
+sys.path.append(os.path.abspath(os.path.join(__file__, "../../../..")))
+
+from v2.lib.resource_op import Config
+from v2.utils import utils
+from v2.utils.log import configure_logging
+from v2.utils.test_desc import AddTestInfo
+
+log = logging.getLogger()
+TEST_DATA_PATH = None
+
+
+def run_command(command, input_string=None):
+    """Runs a shell command and returns the output as a string.
+    If input_string is provided, it will be used as input.
+    """
+    log.info(f"Running command: {command}")
+    try:
+        if input_string:
+            process = subprocess.Popen(
+                command,
+                shell=True,
+                stdin=subprocess.PIPE,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                text=True,
+            )
+            stdout, stderr = process.communicate(input=input_string + "\n")
+            log.info(f"Command output: {stdout.strip()}")
+            if process.returncode != 0:
+                log.error(f"Command failed with error: {stderr.strip()}")
+                sys.exit(1)
+            return stdout.strip()
+        else:
+            process = subprocess.run(
+                command, shell=True, check=True, capture_output=True, text=True
+            )
+            log.info(f"Command output: {process.stdout.strip()}")
+            return process.stdout.strip()
+
+    except Exception as e:
+        log.error(f"Error running command: {e}")
+        sys.exit(1)
+
+
+def check_gc():
```

Comment:
please can you have these functions placed in the reusable.py or reusable folder